model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,75 @@
+groups:
```

Comment:
Is your intent that all of the metrics below should have the requirement level `recommended` (which is the default when not specified), or would we want some of them to be `required`, `conditionally required`, etc...